Readme.md

```diff
@@ -155,6 +157,30 @@ Create a new workflow or add the following step to an existing one that's trigge
 
 You have to provide a JSON object for the `scripts` and `styles-css` inputs, where the property names are the relative paths of the projects you want to inspect, and the values become the`nodejsExtensions` properties in the temporarily generated _package.json_ files used for the linting operation. For more information, check out the workflow inputs [here](.github/workflows/lint.yml).
 
+### Simplified configuration
+
+If the files to be linted are located in the _./Assets/Scripts_ and _./Assets/Styles_ subdirectories, then you can use the simplified comma-separated format:
+
+```yml
+  lint:
+    name: Lint Scripts and Styles
+    uses: Lombiq/NodeJs-Extensions/.github/workflows/lint.yml@dev
+    with:
+      scripts: src/Modules/OrchardCore.Commerce,src/Modules/OrchardCore.Commerce.ContentFields,src/Modules/OrchardCore.Commerce.Payment.Stripe
+      styles-css: src/Modules/OrchardCore.Commerce,src/Modules/OrchardCore.Commerce.Payment
```

Comment:
Do spaces between commas work too? They'd need to, and then change this to reflect that, since that's the standard English spelling.

Reply:
Yes, it works because the code does `$Paths.Split(',') | ForEach-Object { $PSItem.Trim() }` [here](https://github.com/Lombiq/NodeJs-Extensions/blob/6a588f03ca0baf695ba94d5cf7e479758948585c/.github/actions/lint/Invoke-NodeJsLinter.ps1#L75-L76). I've updated the readme.